test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -537,7 +544,9 @@ test
     await t.expect(deviceChallengePollPageObject.getAppLinkContent())
       .contains('If Okta Verify did not open automatically, tap Open Okta Verify.');
     await t.expect(deviceChallengePollPageObject.getPrimaryButtonText()).eql('Open Okta Verify');
-    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(false);
+    if(!userVariables.v3) {
+      await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(false);
```

Comment:
FooterCancelPollingLink and SIgnOutLink are different in v2

Reply:
When you say they are different, are they completely different links? Or is it just a naming difference and functionality is the same, can this conditional be added directly to the function `getFooterCancelPollingLink`?

Reply:
Looks like just the names are different but the functionality is the same in v2.  In v3 they both have the name '`Back to sign in`' so I think this conditional needs to be here in the spec test to check if one exists but not the other for v2 since `getFooterCancelPollingLink` also gets the same Back to sign in link in v3

Reply:
Ahh I see, so it actually looks like there is a parity issue here, it looks like the link should not be `Back to sig in`, based on the implementation [here](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/device/DeviceChallengePollView.js#L61), it looks like it should actually be `Cancel and take me to sign in`.

Reply:
yeah for some reason it looks like in the APP_LINK method during the 'delay', the link is `Cancel and take me to sign in`, then after the delay is finished they update the link the 'Back to sign in'... lol very confusing

Reply:
at this part [here](https://github.com/okta/okta-signin-widget/blob/09fb3ad250f41534934be7f19ccafaac51d2ab0c/src/v2/view-builder/utils/ChallengeViewUtil.js#L121)

Reply:
Oh I see, we should probably create a ticket to implement this logic and either disable these tests or at least add that ticket number here so we can remove this conditional once we get it completely working.

Reply:
okay I may be able to implement a work around